src/openmmio.py

```diff
@@ -1138,6 +1179,8 @@ def evaluate_(self, force=False, dipole=False, traj=False):
         for I in range(len(self.xyz_omms)):
             self.set_positions(I)
             R1 = self.evaluate_one_(force, dipole)
+            # if I == 0:
+            #     printcool_dictionary(energy_components(self.simulation), title='Energy component analysis, shot %i' % I)
```

Comment:
> This should be good to merge if you're OK with the amoeba test beginning to fail (not sure about all the context there so it's your call)

I hadn't checked that the test was failing.  Thanks a lot, and I will look into it.

> "Ever" is a big word, but if it hasn't been working since unit-bearing values in SMIRNOFF started being written as full quantities (around 2019?) then I think we're good without it.

I should clarify, I'm not asking you to commit to never having this feature!  What I meant was, if we ever want to have this feature, we may need to think about the design a bit.  Perhaps FB could split off the unit string of the "initial" evaluated parameter and append it to the computed value, or something like that.

Reply:
Ah, yeah, I'd seen some other code that assumes the new magnitude has the same units as the original value in my PR from earlier. I think it's a bit dangerous (could invite edge cases, eg regularization getting confused if similarly-named fields have different units), so I wouldn't put that logic in more places unless there's a driving use case for it. 

A bigger restructuring might try carrying quantities around internally everywhere, but that's well beyond the scope of a 1.9.5 release. We may experiment with that in the OpenFF fork of forcebalance and could report back if we get it working.